javascriptv3/example_code/ec2/actions/describe-instance-types.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { fileURLToPath } from "url";
+
+// snippet-start:[javascript.v3.ec2.actions.DescribeInstanceTypes]
+import {
+  paginateDescribeInstanceTypes,
+  DescribeInstanceTypesCommand,
+} from "@aws-sdk/client-ec2";
+
+import { client } from "../libs/client.js";
+
+// List at least the first arm64 EC2 instance type available.
```

Comment:
arm64 --> Arm64

Reply:
@lkdavies this is the casing in the responses from the service. Keeping this one.